hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -167,6 +195,33 @@ public void createContract(
         throw new AssertionError("Not implemented");
```

Comment:
I not sure but the difference between this version and the one below is if the createContract is a top level call.  A comment that states the difference would be nice.

Reply:
This overload will set the Hedera properties (`autoRenewId`, staking election, etc.) of the new contract based on the parent contract's properties.

(The overload implemented in this PR instead sets those properties from a `ContractCreateTransactionBody`.)